tests/unit/storage_v3_key_store.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright 2022 Memgraph Ltd.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt; by using this file, you agree to be bound by the terms of the Business Source
+// License, and you may not use this file except in compliance with the Business Source License.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+#include <algorithm>
+#include <string>
+#include <vector>
+
+/**
+ * gtest/gtest.h must be included before rapidcheck/gtest.h!
+ */
+#include <gtest/gtest.h>
+#include <rapidcheck.h>
+#include <rapidcheck/gtest.h>
+
+#include "storage/v3/id_types.hpp"
+#include "storage/v3/key_store.hpp"
+#include "storage/v3/property_value.hpp"
+
+namespace memgraph::storage::v3::test {
+
+RC_GTEST_PROP(KeyStore, KeyStore, (std::vector<std::string> values)) {
```

Comment:
Just to make sure that I understand `RC_GTEST_PROP`.  This is a google macro that effectively uses `RapidCheck` to generate a random list of properties seeded/passed to the `values` parameter. Is that correct?

Reply:
I am not sure if this come with GTest of RapidCheck, but you are right about the functionality of the macro.